README.md

```diff
@@ -25,13 +25,13 @@ The EPL give different awards each season to the players and teams who played, t
 - Golden Boy (Best Sub20 Player)
 - Others to be defined each season
 
-During each match, the EPL has the record of each of the teams involved (Roster of players with name and last name, number, name of the coach), the goals in favor and against before the match & also during the course of the match it records in real time the shots, shots on goal, goals, assists, fouls, cards and goalkeeper saves.
+During each match, the EPL records information of each of the teams involved (Roster of players with name and last name, number, name of the coach), the goals in favor and against before the match. Also, during the course of the match it records in real time the shots, shots on goal, goals, assists, fouls, cards and goalkeeper saves.
```

Comment:
"During each match, the EPL records information of each of the teams involved"
During a match, the EPL records information of each team